examples/eks-msk-opensearch/README.md

```diff
@@ -0,0 +1,113 @@
+* * Broker - A broker is a software services to decouple publisher and consumers. In generic terms it will be our message queue which will store the log messages before these are sent for visulaization. 
+* * MSK - It refers to [Amazon Managed Service for Kafka](https://aws.amazon.com/msk/) ( Our broker in this example) 
+* * [Terraform](https://www.terraform.io/) - It's Hashicorp's tool to define infrastructure as a code .
+* * Amazon Opensearch - OpenSearch(https://aws.amazon.com/what-is/opensearch/) is a distributed, community-driven, Apache 2.0-licensed, 100% open-source search and analytics suite used for a broad set of use cases like real-time application monitoring, log analytics, and website search
+
+In this example, we will showcase how to send your application logs from Amazon Elastic Kubernetes Service (EKS) cluster (https://aws.amazon.com/eks/) to [Amazon OpenSearch](https://aws.amazon.com/what-is/opensearch/) for vislualization. 
+
+
+**We will address following use cases in our solution.**
+
+* Use of a broker to de-couple sending logs directly to OpenSearch so that broker can store logs if the OpenSearch is not available or having  some issues. In this example we will use Amazon MSK to store our logs.
+
+* Fan out logs to multiple destinations. We can use KAFKA consumers or Connectors to send logs from your Pods to different destinations of your choice such as S3 or CloudWatch Logs. So Kafka can act as a Fan-Out source of your application logs.
+
+* Seperate KAFKA topics for each tenant to achieve tenant's log Isolation(logs in seperate topics) and KAFKA sink connector will then send logs to OpenSearch creating unique INDEX per topic, hence giving tenant isolation at OpenSearch also.
+
+To achieve this we will use [fluent-bit](https://fluentbit.io/) to collect logs from your pods. Fluent-bit is a lightweight, and highly scalable logging and metrics processor and forwarder and can be used for kubernetes workloads  to send logs to many supported destniations like CloudWatch Logs, S3 and  OpenSearch. Fluent bit uses following configuration diretives to process incoming logs.
+
+* INPUT to define How to collect data/events.
+* FILTER to modify data to add/remove fields or enrich fields.
+* OUTPUT to configure plugins to forward logs to endpoints like S3,CloudWatch etc. In our example we will use 'KAFKA' OUTPUT plugin.
+
+Here in our case , Fluent Bit will run as a Kubernetes DaemonSet on your EKS cluster to collect logs from Pods.
+
+We have a fluent-bit template in "template" directory, which terraform will use to generate a run-time config from the configuration values of namespaces,brokers etc.
+
+Also note that Fluent bit configuration file has a Lua script FILTER  which is used to set topic names for KAFKA topics such that each tenant/namespace will have a corresponding unique topic "logs_<namespace>". This gives our topics a unique name if KAFKA broker is being used/shared between more than one applications.
```

Comment:
> "logs_<namespace>".

Formatting as `logs_<namespace>` might read/look better

Reply:
Actually its necessary because otherwise markdown formatting will hide the `<namespace>`